playgame.py

```diff
@@ -59,6 +73,10 @@ def run_one_game(args):
 def prep_logger(log_dir, verbose, log_stderr, count):
     formatter = logging.Formatter('[%(asctime)s] %(levelname)8s --- %(message)s ' +
                                   '(%(filename)s:%(lineno)s)',datefmt='%Y-%m-%d %H:%M:%S')
+
+    create_dirs("results",log_dir)
```

Comment:
you shouldn't provide a default directory if it is not provided. If the idea is to keep root clean, let's add some info in the help for log_dir and log_stderr